pyproject.toml

```diff
@@ -150,6 +150,8 @@ Twisted = [
 uvicorn = { extras = ["standard"], version = "^0.12" }
 wordcloud = "^1.8.0"
 youtube-dl = "^2020.7.28"
+django-cachalot = "^2.3.5"
+Werkzeug = "^1.0.1"
```

Comment:
What is Werkzeug for?

Reply:
it's for using `manage.py runserver_plus` (which can be configured to log queries via `--print_sql`)

Reply:
Oh, I see; it's for runserver_plus, right?

Reply:
Should we make this a dev-only dependency?

In production, we use Uvicorn.

Reply:
yeah, I forgot to specify that when adding it.